lms/djangoapps/course_api/tests/test_api.py

```diff
@@ -134,15 +114,23 @@ def _make_api_call(self, requesting_user, specified_user):
         Call the list_courses api endpoint to get information about
         `specified_user` on behalf of `requesting_user`.
         """
-        request = self.request_factory.get('/')
+        request = Request(self.request_factory.get('/'))
```

Comment:
FYI, this was to convert the django Request object created by RequestFactory to a DRF Request object.  Normally this happens behind the scenes when calling a `rest_framework.views.APIView`, but since we're not calling views in this module, I have to convert it manually.  This gives us some extra attributes that we need.  (I think this includes `query_params`, but I don't remember specifically.